internal/impl/kafka/enterprise/integration_test.go

```diff
@@ -776,8 +838,6 @@ input:
       topics: [ %s ]
       consumer_group: migrator_cg
       start_from_oldest: true
-      replication_factor_override: true
-      replication_factor: -1
```

Comment:
The template linter is a bit annoying, since it doesn't check if there are any unrecognised fields... Something to improve in Benthos.